CRM Solution Manager/CRM Solution/App.config

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <appSettings>
-    <add key="OrgServiceUrl" value="https://test.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
-    <add key="DynamicsUserName" value="test@test.onmicrosoft.com" />
-    <add key="DynamicsPassword" value="*****" />
+    <add key="OrgServiceUrl" value="https://msdyn.api.crm8.dynamics.com/XRMServices/2011/Organization.svc" />
+    <add key="DynamicsUserName" value="v-raras@msdyn.onmicrosoft.com" />
```

Comment:
you have checked in sensitive data, remove all this sensitie data

Reply:
committed updated app.config